build-phar.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ ! -f box.phar ]; then
+    wget https://github.com/humbug/box/releases/download/3.0.0-beta.4/box.phar -O box.phar
```

Comment:
How do we trust this box?

Reply:
The releases of BOX are not GPG-signed, so HTTPS should be enough for now at least.
It should be reported upstream as a release blocker probably.

https://github.com/humbug/box/issues/15
https://github.com/humbug/box/issues/14

Reply:
what do you mean?

Reply:
Not sure where my message went, but I'm still unconvinced signing the PHAR has any benefit

Reply:
Well... If the archive is signed with a GPG key of a maintainer, you can validate the signature. But I'm not sure whether it has any real benefit, especially when GitHub releases are the source.

Reply:
Let me put my foil hat on and ask "can we trust GitHub? what if the repo becomes compromised?"

Reply:
Then your releases become compromised too and so do became compromised your `.asc` files there. :roll_eyes:

Reply:
And if you can't trust GitHub you can't trust any package being installed with Composer anyway. It's not an irrelevant issue, but it requires more work to be solved

Reply:
I think that indeed needs exploration: https://github.com/Roave/composer-gpg-verify is just a prototype, but having a trusted keyring on our side would solve this.